python/cuml/decomposition/pca.pyx

```diff
@@ -387,47 +390,52 @@ class PCA:
                                             self.noise_variance_)
         cdef uintptr_t trans_input_ptr = self._get_ctype_ptr(self.trans_input_)
 
+        cdef cumlHandle* h_ = <cumlHandle*><size_t>self.handle.getHandle()
         if not _transform:
             if self.gdf_datatype.type == np.float32:
-                pcaFit(<float*> input_ptr,
-                             <float*> components_ptr,
-                             <float*> explained_var_ptr,
-                             <float*> explained_var_ratio_ptr,
-                             <float*> singular_vals_ptr,
-                             <float*> mean_ptr,
-                             <float*> noise_vars_ptr,
-                             params)
+                pcaFit(h_[0],
```

Comment:
Ccasting here to `<cumlHandle*>` would be necessary to avoid having the double case above of `<cumlHandle*><size_t>`